Utils/Dataflow/pyDKB/common/json_utils.py

```diff
@@ -1,14 +1,56 @@
 """
-Utils to work with JSON (dict) objects.
+Utils to work with JSON objects.
+
+In context of python, JSON [#]_ objects may be considered as structures
+consisting of four types of elements:
+
+- dictionaries,
+- lists,
+- strings,
+- numbers.
+
+DKB project uses JSON for storing various information and transferring it
+between stages. This module contains functions which simplify some aspects
+of retrieving data from JSON objects.
+
+.. [#] https://www.json.org/
+
 """
 
 
 def valueByKey(json_data, key):
     """ Return value by a chain (list) of nested keys.
 
-    Parameters:
+    It is common for JSON objects to contain many layers of dictionaries
+    nested in other dictionaries - this function extracts the data from
```

Comment:
Single hyphen produces short dash, double — long one. In this case I guess you meant to use the long one, right?

Reply:
You are right, thanks. Will fix.